go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/pkg/errors v0.9.1
+	github.com/spf13/cobra v1.0.0
```

Comment:
Run `go mod tidy` and double check that there isn't also an indirect dependency that shows up.

Reply:
There is!

`github.com/inconshreveable/mousetrap v1.0.0 h1:Z8tu5sraLXCXIcARxBp/8cbvlwVa7Z1NHg9XEKhtSvM=`

Reply:
well thanks for trying!